src/infrastructure/figma/figma-client/types.ts

```diff
@@ -0,0 +1,82 @@
+export type GetOAuth2TokenResponse = {
```

Comment:
note: This code has been moved from `infrastructure/figma-client.ts` with almost no changes.